tsconfig-strict.json

```diff
@@ -269,6 +267,8 @@
     "core/templates/domain/story/story-summary.model.spec.ts",
     "core/templates/domain/story/story-validation.service.ts",
     "core/templates/domain/story/story-validation.service.spec.ts",
+    "core/templates/domain/story/storycontents-object.model.ts",
+    "core/templates/domain/story/storycontents-object.model.spec.ts",
```

Comment:
Shouldn't the files be named _story-contents-object_?

Reply:
Yes, you're right!
I'll make the necessary changes.

Reply:
Done!

Reply:
PTAL @vojtechjelinek